pkg/disttask/framework/taskexecutor/slot.go

```diff
@@ -37,6 +37,9 @@ type slotManager struct {
 
 // subtasks inside a task will be run in serial, so they takes task.Concurrency slots.
 func (sm *slotManager) alloc(task *proto.Task) {
+	if task.State != proto.TaskStateRunning {
+		return
+	}
```

Comment:
shouldn't we fix the case that disttask is still `reverting`, but ddl mark it as `rollback done`

it's ok for reverting task to take slots, as it's transient normally

Reply:
I am working on it